src/compiler/types.ts

```diff
@@ -1007,6 +1013,10 @@ namespace ts {
         expression: LeftHandSideExpression;
         typeArguments?: NodeArray<TypeNode>;
     }
+    export interface SupportedExpressionWithTypeArguments extends ExpressionWithTypeArguments {

```

Comment:
What properties make it supported or unsupported? That's what should go in the name.

---
In reply to: [72693405](https://github.com/Microsoft/TypeScript/pull/10013#discussion_r72693405) [](ancestors = 72693405)